### PR TITLE
Phase1

### PR DESCRIPTION
**Pull request by omoskalyov**
Please review my implementation of Phase1 task from AQA Python course.

### CODE REVIEW

File: phase1/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pytest==3.7.3
```
Comment (andynati) [inline phase1/requirements.txt:1]: Move this file from phase1 to root direcory for fix CircleCI failed step `Restoring Cache`

File: phase1/src/fibonacci/fibonacci.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+def generateFibonacci(length):
+    """ The function generates Fibonacci sequence.
+    Params:
+        length - number of numbers
+    Output:
+        Fibonacci number
+    """
```
Comment (andynati) [inline phase1/src/fibonacci/fibonacci.py:9]: We expecting a sequence (as a list) as return here. But a single number is ok

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    working_directory: ~/aqapython
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "requirements.txt" }}
+          - v1-dependencies-
```
Comment (andynati) [inline .circleci/config.yml]: looks like extra line.

**Review by andynati — CHANGES_REQUESTED** [review]

File: phase1/requirements.txt (hunk 1 above)
Comment (omoskalyov) [reply phase1/requirements.txt:1]: Fixed another way:
specified correct location of this file in .circleci/config.yml (just want to keep files related to a particular phase in a single dir)

File: phase1/src/fibonacci/fibonacci.py (hunk 2 above)
Comment (omoskalyov) [reply phase1/src/fibonacci/fibonacci.py:9]: Leaving as is (if not important)

File: .circleci/config.yml (hunk 3 above)
Comment (omoskalyov) [reply .circleci/config.yml]: Don't have deep knowledge of caching mechanism in circleci but this extra line is needed for fallback caching mechanism (see here https://circleci.com/docs/2.0/caching/).

**Review by andynati — APPROVED** [review]

**Pull request merged**